backend/emails/email_sending.py

```diff
@@ -0,0 +1,25 @@
+from email.message import EmailMessage
+import smtplib
+import os
+
+LOGIN = "contest.platform1@gmail.com"
+
+
+def send_email(receiver, subject, body):
+    password = os.getenv("PASSWORD", None)
```

Comment:
Make this env more descriptive. Some tests would be appreciated, and a fixture that mocks sending emails